MapboxMobileEventsTests/MMEEventTests.mm

```diff
@@ -129,6 +135,103 @@
             exceptionEventWithAllInfo should_not be_nil;
         });
     });
+
+    context(@"eventWithAttributes:error:", ^{
+        it(@"should not init with invalid attributes", ^{
+            NSError *error = nil;
```

Comment:
Kind of a nit but we could move `NSError *error = nil;` into a [`beforeEach` block](https://github.com/cedarbdd/cedar/wiki/Writing-specs#extracting-common-setup---beforeeach) to fire before each assert/`it`.

Reply:
I like it more that each test is self-contained, overall this structure of that context might need some work